latch_cli/centromere/ctx.py

```diff
@@ -142,6 +149,10 @@ def __init__(
                 )
                 from ..snakemake.utils import load_snakemake_metadata
 
+                name_path = pkg_root / latch_constants.pkg_workflow_name
+                if name_path.exists():
+                    self.workflow_name = name_path.read_text().strip()
```

Comment:
?

Reply:
i think this got fucked up when i merged into master